src/main.c

```diff
@@ -1,7 +1,7 @@
 /*
  *      main.c - this file is part of Geany, a fast and lightweight IDE
  *
- *      Copyright 2014 Matthew Brush <mbrush@codebrainz.ca>
+ *      Copyright 2005 The Geany contributors
```

Comment:
That date change is kinda odd

Reply:
This is because: https://github.com/geany/geany/commit/d33758da926ca124db674240c431b8b07bb0e683#diff-803c5170888b8642f2a97e5e9423d399L5. `src/main.c` was one of the very first files ever.

Reply:
yeah but the old main.c became the new libmain.c, didn't it?  anyway, it doesn't matter, and yes the file was there and did contain something reasonably similar.